var/spack/repos/builtin/packages/gtkplus/package.py

```diff
@@ -49,6 +49,7 @@ class Gtkplus(MesonPackage):
     depends_on('cups', when='+cups')
 
     patch('no-demos.patch', when='@2:2.99')
+    patch('gcc10.patch', when='%gcc@10:')
```

Comment:
Can you add a comment on top of the patch to explain why it is needed? Maybe link the issue you have in the description. Bonus points if you can retrieve the patch from `https` instead of storing it alongside `package.py`.

Reply:
@joequant